app/controllers/TimeoutController.scala

```diff
@@ -36,7 +36,8 @@ class TimeoutController @Inject() (views:      Views,
   val killSession: Action[AnyContent] = Action { implicit request =>
     appLogger.info("Kill session")
 
-    Ok(views.delete_answers(controllers.routes.TimeoutController.signInAgain.url)).withNewSession
+    val signInAgainCall = Call("GET", controllers.routes.TimeoutController.signInAgain.url)
+    Ok(views.delete_answers(signInAgainCall)).withNewSession
```

Comment:
`controllers.routes.TimeoutController.signInAgain` should be of type `Call` so you could just pass that in

Reply:
Yes, have fixed